extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -256,6 +257,9 @@ public function getWikiVariables() {
 
 		$this->response->setVal( 'data', $wikiVariables );
 		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+
+		//cache wikiaVariables for 1 minute		
+		$this->response->setCacheValidity( self:: WIKIA_VARIABLES_CACHE_TTL);
```

Comment:
invalidating cache will require quite a lot of effort in this case. Can I find you tomorrow and can we talk about this?